### PR TITLE
Format Management csproj with updated codeformatter

### PR DESCRIPTION
**Pull request by iSazonov**
## PR Summary

Related #4708.

Format Microsoft.PowerShell.Commands.Management.csproj by codeformatter with default options.

The codeformatter is used that I compiled with newest Roslyn packages (version 2.8.2).

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs [hunk 1]
```diff
@@ -215,13 +215,13 @@ protected override void StopProcessing()
...
-// get
+        public virtual string[] Include
+        {
+            get;
             set;
-// set
         } = new string[0];
 
-// Include
+        // Include
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs]: Perhaps delete this comment

File: src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs [hunk 2]
```diff
@@ -233,13 +233,15 @@ protected override void StopProcessing()
         /// the base class to simplify the creation of the CmdletProviderContext.
         /// </remarks>
         ///
-        public virtual string[] Exclude { get;
-// get
+        public virtual string[] Exclude
+        {
+            get;
+            // get
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs]: Delete

File: src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs [hunk 3]
```diff
@@ -233,13 +233,15 @@ protected override void StopProcessing()
...
             set;
-// set
+            // set
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs]: delete

File: src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs [hunk 4]
```diff
@@ -233,13 +233,15 @@ protected override void StopProcessing()
...
         } = new string[0];
 
-// Exclude
+        // Exclude
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs]: delete

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs [hunk 5]
```diff
@@ -233,13 +233,13 @@ protected override void StopProcessing()
...
-// get
+        public virtual string[] Exclude
+        {
+            get;
             set;
-// set
         } = new string[0];
 
-// Exclude
+        // Exclude
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs]: Delete

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

**Comment by iSazonov** [comment]
@SteveL-MSFT You comments was addressed

**Review by SteveL-MSFT — APPROVED** [review]

**Review by daxian-dbw — APPROVED** [review]
Good work! Thanks @iSazonov

**Pull request merged**